react.js

```diff
@@ -44,8 +44,6 @@ const customRender = (node, options = {}) => {
     messages: messages,
   }
 
-  const Provider = options.MockedProvider || MockedProvider
```

Comment:
When we did that we also changed the typings. Can you fix them too?

Reply:
done